### PR TITLE
feat: improve walkthrough

### PR DESCRIPTION
**Pull request by anchenyi**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/30591407

image
image

### CODE REVIEW

File: packages/vscode-extension/media/walkthroughs/get-started/create/create.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 | Step summary        | Where it's located              |
 |----------|----------------|
-| 1. Click "Create a New App" | ![Image1](step1.png) |
-| 2. Select Bot to create a new bot project | ![Image2](step2.png) |
-| 3. Ensure that Chat Notification Message is selected as the App feature that you want to build in your app. | ![Image3](step3.svg) |
+| 1. Click "Create a New App". | ![Image1](step1.png) |
+| 2. Select Bot to create a new bot project. | ![Image2](step2.png) |
+| 3. Make sure your select Chat Notification Message as the App feature that you want to build in your app. | ![Image3](step3.svg) |
```
Comment (supkasar) [inline packages/vscode-extension/media/walkthroughs/get-started/create/create.md]: change 'your' to 'you' i.e. Make sure you select...

File: packages/vscode-extension/media/walkthroughs/get-started/create/create_with_chat.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 | Step summary        | Where it's located              |
 |----------|----------------|
-| 1. Click "Create a New App" | ![Image1](step1_with_chat.png) |
-| 2. Select Bot to create a new bot project | ![Image2](step2_with_chat.png) |
-| 3. Ensure that Chat Notification Message is selected as the App feature that you want to build in your app. | ![Image3](step3.svg) |
+| 1. Click "Create a New App". | ![Image1](step1_with_chat.png) |
+| 2. Select Bot to create a new bot project. | ![Image2](step2_with_chat.png) |
+| 3. Make sure your select Chat Notification Message as the App feature that you want to build in your app. | ![Image3](step3.svg) |
```
Comment (supkasar) [inline packages/vscode-extension/media/walkthroughs/get-started/create/create_with_chat.md]: Same here.

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -559,24 +559,24 @@
...
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.description": "To extend Microsoft 365 Copilot with declarative agent, in addition to Node.js and NPM requirements, you will also need a valid license to access Microsoft 365 Copilot. [Checkout this document for obtaining developer environment.](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#prerequisites)\n[Check Copilot License](command:fx-extension.checkCopilotAccess?%5B%22WalkThrough%22%5D)",
+  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.title": "Set up your environment",
+  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.description": "Set up your environment to build a declarative agent for Microsoft 365 Copilot. Ensure you have Node.js, NPM, and a valid Microsoft 365 Copilot license. Learn how to get a developer environment here. [Learn how to get a developer environment.](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#prerequisites)\n[Check Copilot License](command:fx-extension.checkCopilotAccess?%5B%22WalkThrough%22%5D)",
```
Comment (supkasar) [inline packages/vscode-extension/package.nls.json]: Please remove the line - Learn how to get a developer environment here. It's a duplicate of hyperlinked line.

File: packages/vscode-extension/media/walkthroughs/get-started/create/create.md (hunk 1 above)
Comment (anchenyi) [reply packages/vscode-extension/media/walkthroughs/get-started/create/create.md]: Updated. Thanks!

File: packages/vscode-extension/media/walkthroughs/get-started/create/create_with_chat.md (hunk 2 above)
Comment (anchenyi) [reply packages/vscode-extension/media/walkthroughs/get-started/create/create_with_chat.md]: Updated. Thanks!

File: packages/vscode-extension/package.nls.json (hunk 3 above)
Comment (anchenyi) [reply packages/vscode-extension/package.nls.json]: Updated. Thanks!

**Review by tecton — APPROVED** [review]

**Review by supkasar — APPROVED** [review]

**Pull request merged**